### PR TITLE
Initial commit of the awards playbook and awards list for 2020

### PR DESCRIPTION
**Pull request by castrojo**
Deb from CNCF has reviewed this so those steps are correct. 

I currently do not know who got previous awards, hopefully @parispittman does. Also not sure of the formatting for the awards page, I know we should have a great page with pics of the awards, but we need more discussion on that, in the meantime I'll add it as a simple list.

### CODE REVIEW

**Comment by mrbobbytables** [comment]
There are other details I'd like to see, but we can do that in a follow up.
/lgtm

File: events/awards/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+### SIG Cluster Lifecycle
+
+- Ciprian Hacman
+- John Gardiner Myers
+- Ole Markus
+
+## SIG Documentation
+
+- Tim Bannister
+- Quiming Teng
```
Comment (tengqm) [inline events/awards/README.md:49]:
```suggestion
- Qiming Teng
```

**Comment by nikhita** [comment]
/approve

/hold
pending @tengqm's comment - https://github.com/kubernetes/community/pull/5519#discussion_r577291362

**Review by nikhita — APPROVED** [review]
/lgtm
/approve

/woof

**Pull request merged**